src/components/Menu/index.tsx

```diff
@@ -6,27 +6,27 @@ import Popper from '../Popper'
 import Item from './Item'
 
 const bottomStyles = (theme: Theme) => css`
-  box-shadow: 0 -1px 5px 3px ${transparentize(0.85, theme.colorsDeprecated.shadow)};
+  box-shadow: 0 -1px 5px 3px ${transparentize(0.85, theme.colors.neutral.background)};
   &:after,
   &:before {
     bottom: 100%;
   }
   &:after {
-    border-bottom-color: ${theme.colorsDeprecated.white};
+    border-bottom-color: ${theme.colors.primary.background};
   }
   &:before {
     border-bottom-color: rgba(165, 165, 205, 0.4);
   }
 `
 const topStyles = (theme: Theme) => css`
   box-shadow: 0 1px 5px 3px
-    ${transparentize(0.85, theme.colorsDeprecated.shadow)};
+    ${transparentize(0.85, theme.colors.neutral.background)};
   &:after,
   &:before {
     top: 100%;
   }
   &:after {
-    border-top-color: ${theme.colorsDeprecated.white};
+    border-top-color: ${theme.colors.primary.background};
```

Comment:
```suggestion
    border-top-color: ${theme.colors.primary.backgroundWeak};
```

weak = background light

Reply:
PR has been merge you can rebase :)